src/shx.js

```diff
@@ -5,7 +5,18 @@ import help from './help';
 import { CMD_BLACKLIST, EXIT_CODES, CONFIG_FILE } from './config';
 import { printCmdRet } from './printCmdRet';
 import path from 'path';
-import pathExists from 'path-exists';
+import fs from 'fs';
+import objAssign from 'es6-object-assign';
+objAssign.polyfill(); // modifies the global object
+
+const pathExistsSync = (filePath) => {
```

Comment:
We were originally using that, but it only works on node v4+ so I had to switch to a different implementation. This is based off [this comment](https://github.com/nodejs/node/issues/1592#issuecomment-98392899) on a thread regarding the deprecation.